.template/addons/rspec/Gemfile.rb

```diff
@@ -0,0 +1,33 @@
+############################
+# Group: :development, :test
+############################
+insert_into_file 'Gemfile', after: /gem 'fabrication'.*\n/ do
+  <<~RUBY.indent(2)
+    # Testing
```

Comment:
```suggestion
  <<~RUBY.indent(2)
  
    # Testing
```

We can add an empty line here to make it consitent :)

|Current | Expected|
|---|---|
|![image](https://user-images.githubusercontent.com/77609814/174273270-141e649d-35b5-4607-bd2e-1b1ef79019f7.png)|![image](https://user-images.githubusercontent.com/77609814/174273354-7ab74c6f-ad07-44ba-8012-1b8c0966091d.png)|